packages/01-host/webpack.config.js

```diff
@@ -28,14 +29,19 @@ module.exports = {
         options: {
           presets: [require.resolve("@babel/preset-react")]
         }
+      },
+      {
+        parser: {
+          system: false
+        }
       }
-    ]
+    ],
   },
 
   plugins: [
     new ModuleFederationPlugin({
       name: "app_one",
-      library: { type: "var", name: "app_one" },
+      library: { type: "system" },
```

Comment:
oh, silly me. removing the name removes the named portion of the module. that makes sense.

Reply:
Yeah haha I ran into the same problem with the named register but then found this way of removing the name